akka-docs/rst/java/code/docs/http/javadsl/server/directives/CustomHttpMethodExamplesTest.java

```diff
@@ -67,8 +68,8 @@ public void testComposition() throws InterruptedException, ExecutionException {
     CompletionStage<HttpResponse> response = http.singleRequest(request, materializer);
     //#customHttpMethod
 
-    assertResult(StatusCodes.OK, response.toCompletableFuture().get().status());
```

Comment:
assertResult now takes an org.scalactic.Position implicit argument, which is likely not working here as it relies on macros as far as I can tell.